### PR TITLE
PAT-473: Change Helm chart repository from cammis to hello-cammis

### PR DESCRIPTION
**Pull request by krenshaw2018**
This simple change is to point the requirements.yml file from @cammis to @hello-cammis.

### CODE REVIEW

File: helm/hello-cammis-ui/requirements.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 dependencies:
   - name: hello-cammis-data
     version: 2.2.1
-    repository: "@cammis"
+    repository: "@hello-cammis"
```
[the remark this answers is not in the document]
Comment (krenshaw2018) [reply helm/hello-cammis-ui/requirements.yaml]: Thanks, I had thought hello-cammis would use a new Helm repo sorry. This has been reverted in both hello-cammis-ui and hello-cammis-api.

**Comment by krenshaw2018** [comment]
This re-review request is to OK the MASTER_PLAN override. Thanks.

File: .circleci/config.yml [hunk 2]
```diff
@@ -169,7 +169,7 @@ jobs:
             # Local env vars are not shared between "run" stanzas
             if [ "${CIRCLE_BRANCH}" ] ; then
               REVIEW_ENV_NAME="review_${CIRCLE_PROJECT_REPONAME}_br_${CIRCLE_BRANCH}"
-              #MASTER_CHART_NAME=cammis  # default is cammis
+              MASTER_CHART_NAME=hello-cammis  # default is cammis
```
Comment (gubbin) [inline .circleci/config.yml]: Looks good

Comment (krenshaw2018) [reply .circleci/config.yml]: Hello @gubbin can you please Approve this one so I can get it closed out? Thanks much Sir!

**Pull request merged**